docs/application/native/guides/device/resource-monitor.md

```diff
@@ -0,0 +1,302 @@
+        for (i = 0; i < res_count; i++) {
+            cpu_ids[i] = resource_monitor_create_resource(monitor_id, RESOURCE_MONITOR_TYPE_CPU);
+            if (cpu_ids[i] < 0) {
+                printf("failed to create resource: %d", RESOURCE_MONITOR_TYPE_CPU);
+                goto error_cleanup;
+            }
+
+            error = resource_monitor_set_resource_attr(monitor_id, cpu_ids[i], RESOURCE_MONITOR_CPU_CUR_FREQ);
+            if (error < 0) {
+                printf("failed to set resource atrribute [%d]\n", RESOURCE_MONITOR_SYSTEM_ATTR_POSSIBLE_CPU);
+                goto error_cleanup;
+            }
+
+            error = resource_monitor_set_resource_ctrl(monitor_id, cpu_ids[i], RESOURCE_MONITOR_CTRL_CLUSTER_ID, i);
+            if (error < 0) {
+                printf("failed to set resource control [%d]\n", RESOURCE_MONITOR_SYSTEM_ATTR_POSSIBLE_CPU);
+                goto error_cleanup;
+            }
+        }
+
+        /* Update attributes */
+        error = resource_monitor_update(monitor_id);
+        if (error < 0) {
+            printf("failed to update monitoring data\n");
+            i--;
+            goto error_cleanup;
+        }
+
+        for (i = 0; i < res_count; i++) {
+            error = resource_monitor_get_value_int(monitor_id, res_id, RESOURCE_MONITOR_CPU_CUR_FREQ, &results[i]);
+            if (error < 0) {
+                printf("failed to get attribute data\n");
+                goto error_cleanup;
+            }
+        }
+
+        *num_cpu = res_count;
+
+        error = RESOURCE_MONITOR_ERROR_NONE;
+
+    error_cleanup:
+        for (;i >= 0; i--)
+            resource_monitor_delete_resource(monitor_id, cpu_ids[i]);
+
+        free(cpu_ids);
+
+        return error;
+    }
+
+    ```
+
+## Resource Monitor Keys
+
+<a name="resource_types"></a>
+
+The following table lists the available resource types, which are part of the `resource_monitor_type_e` enumeration:
+
+**Table: Resource Type Keys**
+
+ | Keys                                           | Description                               |
+ |------------------------------------------------|-------------------------------------------|
+ | `RESOURCE_MONITOR_TYPE_CPU`                    | Resource type for CPU clusters.           |
+ | `RESOURCE_MONITOR_TYPE_BUS`                    | Resource type for BUS devices.            |
+ | `RESOURCE_MONITOR_TYPE_GPU`                    | Resource type for GPU devices.            |
+ | `RESOURCE_MONITOR_TYPE_MEMORY`                 | Resource type for main memory.            |
+ | `RESOURCE_MONITOR_TYPE_BATTERY`                | Resource type for battery.                |
+ | `RESOURCE_MONITOR_TYPE_DISPLAY`                | Resource type for display devices.        |
+ | `RESOURCE_MONITOR_TYPE_SYSTEM'`                | Resource type for system overall status   |
+ | `RESOURCE_MONITOR_TYPE_DISK`                   | Resource type for disk devices.           |
```

Comment:
The change has been suggested to maintain consistency with other table contents in this documents.
```suggestion
 | `RESOURCE_MONITOR_TYPE_CPU`                    | Resource type for CPU clusters           |
 | `RESOURCE_MONITOR_TYPE_BUS`                    | Resource type for BUS devices            |
 | `RESOURCE_MONITOR_TYPE_GPU`                    | Resource type for GPU devices            |
 | `RESOURCE_MONITOR_TYPE_MEMORY`                 | Resource type for main memory            |
 | `RESOURCE_MONITOR_TYPE_BATTERY`                | Resource type for battery                |
 | `RESOURCE_MONITOR_TYPE_DISPLAY`                | Resource type for display devices        |
 | `RESOURCE_MONITOR_TYPE_SYSTEM'`                | Resource type for system overall status   |
 | `RESOURCE_MONITOR_TYPE_DISK`                   | Resource type for disk devices           |
```